Userland/Libraries/LibC/scanf.cpp

```diff
@@ -413,7 +413,7 @@ extern "C" int vsscanf(const char* input, const char* format, va_list ap)
         // Parse width specification
         [[maybe_unused]] int width_specifier = 0;
         if (format_lexer.next_is(isdigit)) {
-            auto width_digits = format_lexer.consume_while([](char c) { return isdigit(c); });
+            auto width_digits = format_lexer.consume_while(isdigit);
```

Comment:
`isdigit` returns an `int` which means it won't be accepted as a `Function<bool(char)>` - or did this interface change?

Reply:
It did change: instead of taking a `Function<bool(char)>` we have a templated version of `consume_while()`-like methods. See it [there](https://github.com/SerenityOS/serenity/blob/d65fa91bc49d2db5c2ada06584f240f2b09ba7c3/AK/GenericLexer.h#L180)

Reply:
Ah nice : )